src/services/PromotionService.ts

```diff
@@ -123,17 +124,17 @@ class PromotionService {
       promotionQueryDTO.discountType = discount;
     }
 
-    // if (sort) {
-    //   const queryParams: { [paramKey: string]: string } = { sort };
-    //   if (sort === Sort.Distance) {
-    //     const {
-    //       coords: { latitude, longitude },
-    //     } = await LocationService.GeolocationPosition.getCurrentLocation();
-    //     queryParams.lat = `${latitude}`;
-    //     queryParams.lon = `${longitude}`;
-    //   }
-    //   promotionQueryDTO.push(queryParams);
-    // }
+    if (sort) {
+      promotionQueryDTO.sort = sort;
```

Comment:
The sort by recency seems to be sorting the wrong direction (showing the oldest ones at the top)

Reply:
Made a PR https://github.com/ubclaunchpad/PromoPal-backend/pull/137

Reply:
You can use your admin privileges for the BE to merge this

Reply:
Thank you!!